src/html.pre.js

```diff
@@ -55,30 +56,23 @@ function pre(context) {
     }
     $(document.body).children().wrapAll(div);
   }
+  */
```

Comment:
Why not removing the code ?

Reply:
That was the goal, yes (it's still a draft) 😉